Tools/ardupilotwaf/boards.py

```diff
@@ -169,6 +169,9 @@ def configure_env(self, cfg, env):
             '-Wfatal-errors',
             '-Wno-trigraphs',
         ]
+        env.CFLAGS += [
+            '-std=c99'
```

Comment:
Any reason for not using c11 (or even gnu11)?

Reply:
fixed